cmd/api-endpoint/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/exit"
+	"github.com/edulinq/autograder/internal/log"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter(s) for the endpoint" arg:"" optional:""`
+	Table      bool     `help:"Output data as a TSV." default:"false"`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Execute an API request to the specified endpoint."),
+	)
+
+	err := config.HandleConfigArgs(args.ConfigArgs)
+	if err != nil {
+		log.Fatal("Failed to load config options.", err)
+	}
+
+	var description core.EndpointDescription
+
+	describe := api.Describe(*api.GetRoutes())
```

Comment:
Bad name.
"describe" is a verb, variable names (especially if they are not a function) are usually nouns.